.github/workflows/publish.yml

```
@@ -38,3 +38,5 @@ jobs:
           branch: gh-pages
           folder: _site
           single-commit: true
+          git-config-name: 'github-actions[bot]'
+          git-config-email: 'github-actions[bot]@users.noreply.github.com'
```

Small request, could we omit the quotes here?

```suggestion
          git-config-name: github-actions[bot]
          git-config-email: github-actions[bot]@users.noreply.github.com
```